src/wovn_interceptor.php

```diff
@@ -20,10 +21,15 @@
 use Wovnio\Wovnphp\Utils;
 use Wovnio\Wovnphp\API;
 use Wovnio\Wovnphp\Diagnostics;
+use Wovnio\Wovnphp\RequestOptions;
 use \Wovnio\Wovnphp\CookieLang;
 
 // GET STORE AND HEADERS
 list($store, $headers) = Utils::getStoreAndHeaders($_SERVER, $_COOKIE);
+// RequestOptions here?
+
+$queryString = parse_url($headers->getEnv()['REQUEST_URI'])['query'];
```

Comment:
yes, request options here but you can use $_SERVER directly instead of headers->getEnv

Reply:
well, I guess we could make $headers->query public as well, regrettably